pages/sponsors/index.tsx

```diff
@@ -1,19 +1,55 @@
 import Head from 'next/head';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { GetServerSideProps } from 'next';
+import { RequestHelper } from '../../lib/request-helper';
+import SponsorCard from '../../components/SponsorCard';
+import SponsorHeader from '../../components/SponsorHeader';
 
 /**
  * The sponsors page.
  *
  * Landing: /sponsors
  */
-export default function Sponsors() {
+export default function Sponsors(props: { sponsorCard: Sponsor[] }) {
+  const [sponsor, setSponsor] = useState<Sponsor[]>([]);
+
+  useEffect(() => {
+    setSponsor(props.sponsorCard);
+    //organize challenges in order by rank given in firebase
+  });
+
   return (
-    <div className="flex flex-col flex-grow">
+    <div className="flex flex-col flex-grow bg-indigo-200">
       <Head>
         <title>HackPortal - Sponsors</title>
         <meta name="description" content="HackPortal's Sponsors Page" />
       </Head>
-      <section id="subheader" className="p-4"></section>
+      <SponsorHeader
```

Comment:
Any way to make this dynamic?

Reply:
I think we can create a document on backend to store those data, and then fetch them at `getServerSideProps`

Reply:
Let's do that in 1.1 - we'll need it moving forward anyways to have the information display sitewide